src/core/int128.d

```diff
@@ -18,10 +18,21 @@ alias I = long;
 alias U = ulong;
 enum Ubits = uint(U.sizeof * 8);
 
-align(16) struct Cent
+version (x86)
 {
-    U lo;      // low 64 bits
-    U hi;      // high 64 bits
+    struct Cent
+    {
+        U lo;      // low 64 bits
+        U hi;      // high 64 bits
+    }
+}
+else
+{
+    align(16) struct Cent
+    {
+        U lo;      // low 64 bits
+        U hi;      // high 64 bits
+    }
```

Comment:
```
version (X86_64) private enum Cent_alignment = 16;
else             private enum Cent_alignment = (size_t.sizeof * 2);

align(Cent_alignment) struct Cent
{
}
```
X86_64 is a special case because of X32, otherwise most other targets uniformly have pointer size == word size.